app.js

```diff
@@ -0,0 +1,93 @@
+const form = document.querySelector('#new-task-form');
+const button = document.querySelector('#button');
+const list =  document.querySelector('#task-list');
+const field = form['new-task-input'];
+
+
+const todoList = [];
```

Comment:
Generally we try to avoid words like `list`, `collection` in variable naming.
Use pluralisation to denote that a variable contains a collection of items.
```suggestion
const todos = [];
```